include/fpm/ios.hpp

```diff
@@ -737,4 +737,44 @@ std::basic_istream<CharT, Traits>& operator>>(std::basic_istream<CharT, Traits>&
 
 }
 
+#if __cplusplus >= 202002L
+#   include <version>
+#endif
```

Comment:
I think this `#endif` should be moved to the end, because as you said, `__cpp_lib_format` won't work if we don't include `<version>`.